pipework

```diff
@@ -412,7 +412,11 @@ elif [ "$IFTYPE" = tc ] ; then
   ip netns exec "$NSPID" tc "$@"
 else
   # Otherwise, run normally.
-  ip link set "$GUEST_IFNAME" netns "$NSPID"
+  if iw dev | awk '$1=="Interface"{print $2}' | grep $GUEST_IFNAME > /dev/null; then
```

Comment:
We should first check if `iw` exists (it doesn't exist on most of my servers) to avoid printing a spurious error message

Reply:
I added the check of iw presence ;-)